src/Domain/Common/Base/Entity.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SSW.CleanArchitecture.Domain.Common.Base;
+
+/// <summary>
+/// Entities have an ID and a lifecycle (i.e. created, modified, and deleted)
+/// They can be created within the domain, but not externally.
+/// Enforce business rules (i.e. invariants)
+/// </summary>
+public abstract class Entity<TId> : AuditableEntity
```

Comment:
This feels backwards that all Entities are Auditable, but not all AuditableEntities are Entities?

Reply:
As per our conversation, we've decided to leave this as is for now.  We may re-evaluation in future.

Reply:
As discussed with the team, we will create an issue to look into this.